homeassistant/components/resolution_center/issue_handler.py

```diff
@@ -1,12 +1,76 @@
 """The resolution center integration."""
 from __future__ import annotations
 
+from typing import Any
+
 from awesomeversion import AwesomeVersion, AwesomeVersionStrategy
 
+from homeassistant import data_entry_flow
 from homeassistant.core import HomeAssistant, callback
+from homeassistant.exceptions import HomeAssistantError
+from homeassistant.helpers.integration_platform import (
+    async_process_integration_platforms,
+)
 
+from .const import DOMAIN
 from .issue_registry import async_get as async_get_issue_registry
-from .models import IssueSeverity
+from .models import IssueSeverity, ResolutionCenterFlow, ResolutionCenterProtocol
+
+
+class ResolutionCenterFlowManager(data_entry_flow.FlowManager):
+    """Manage resolution center flows."""
+
+    async def async_create_flow(
+        self,
+        handler_key: Any,
+        *,
+        context: dict[str, Any] | None = None,
+        data: dict[str, Any] | None = None,
+    ) -> ResolutionCenterFlow:
+        """Create a flow. platform is a resolution center module."""
+        if "platforms" not in self.hass.data[DOMAIN]:
+            await async_process_resolution_center_platforms(self.hass)
+
+        platforms: dict[str, ResolutionCenterProtocol] = self.hass.data[DOMAIN][
+            "platforms"
+        ]
+        platform = platforms[handler_key]
+
+        assert data and "issue_id" in data
+        return await platform.async_create_fix_flow(self.hass, data["issue_id"])
```

Comment:
Maybe we should check that the issue exists and is tagged as fixable before calling the platform?

Reply:
Fixed in 7fa007b